extractor/src/test/java/org/schabi/newpipe/extractor/services/media_ccc/MediaCCCStreamExtractorTest.java

```diff
@@ -42,26 +44,43 @@ public static void setUp() throws Exception {
         @Override public String expectedId() { return ID; }
         @Override public String expectedUrlContains() { return URL; }
         @Override public String expectedOriginalUrlContains() { return URL; }
-
         @Override public StreamType expectedStreamType() { return StreamType.VIDEO_STREAM; }
         @Override public String expectedUploaderName() { return "gpn18"; }
         @Override public String expectedUploaderUrl() { return "https://media.ccc.de/c/gpn18"; }
         @Override public List<String> expectedDescriptionContains() { return Arrays.asList("SSH-Sessions", "\"Terminal Multiplexer\""); }
         @Override public long expectedLength() { return 3097; }
         @Override public long expectedViewCountAtLeast() { return 2380; }
         @Nullable @Override public String expectedUploadDate() { return "2018-05-11 00:00:00.000"; }
-        @Nullable @Override public String expectedTextualUploadDate() { return "2018-05-11T02:00:00.000+02:00"; }
-        @Override public long expectedLikeCountAtLeast() { return -1; }
-        @Override public long expectedDislikeCountAtLeast() { return -1; }
-        @Override public boolean expectedHasRelatedStreams() { return false; }
-        @Override public boolean expectedHasSubtitles() { return false; }
-        @Override public boolean expectedHasFrames() { return false; }
-        @Override public List<String> expectedTags() { return Arrays.asList("gpn18", "105"); }
-        @Override public int expectedStreamSegmentsCount() { return 0; }
+        @Nullable @Override public String expectedTextualUploadDate() {
+            return "2018-05-11T02:00:00.000+02:00";
+        }
+        @Override public long expectedLikeCountAtLeast() {
+            return -1;
+        }
+        @Override public long expectedDislikeCountAtLeast() {
+            return -1;
+        }
+        @Override public boolean expectedHasRelatedStreams() {
+            return false;
+        }
+        @Override public boolean expectedHasSubtitles() {
+            return false;
+        }
+        @Override public boolean expectedHasFrames() {
+            return false;
+        }
+        @Override public List<String> expectedTags() {
+            return Arrays.asList("gpn18", "105");
+        }
+        @Override public int expectedStreamSegmentsCount() {
+            return 0;
+        }
+        @Override public Locale expectedLanguageInfo() {
+            return new Locale("de");
+        }
```

Comment:
I am not sure about these lines and the other test files. these changes make the tests longer and thus harder to read, because finding the important info takes longer.

IMO, that can be kept in one line as long as that lines does not get way too long.
CC @Stypox @XiangRongLin

Reply:
I'm really confused, they are in one line in intellij, but not on GitHub / other editors
fixed in the last push
i agree it's more readable